include/ignition/fuel_tools/FuelClient.hh

```diff
@@ -348,6 +348,21 @@ namespace ignition
                   const ignition::fuel_tools::ModelIdentifier &_model,
                   const std::vector<std::string> &_headers);
 
+      /// \brief Update a model using a PATCH request.
+      ///
+      /// Model fields that are patched by this function:
+      ///   * private
```

Comment:
Is this correct? The files are also patched, right?

Reply:
Update in e3ae7d852192597e06022dbb5a3e9a03cdca5485